### PR TITLE
Update android allowBackup to false

### PR DESCRIPTION
**Pull request by SMJ93**
Update android allowBackup to false so it is compatible with the latest version of React Native:

https://github.com/ncuillery/rn-diff/compare/rn-0.53.3...rn-0.54.0

### CODE REVIEW

File: android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.appboy.reactbridge">
 
-  <application android:allowBackup="true"
+  <application android:allowBackup="false"
```
Comment (Bucimis) [inline android/src/main/AndroidManifest.xml]: @SMJ93 feel free to remove this attribute altogether, braze has no interest in specifying it

**Comment by Bucimis** [comment]
@SMJ93 merged, thanks for your contribution! Would you like us to release the change to npm or will you use the source for now?

**Comment by SMJ93** [comment]
@Bucimis thanks Bucimis, we can use the source

**Comment by Bucimis** [comment]
@SMJ93 great, thanks again. The next npm release will contain the fix as well.

**Pull request merged**